public/js/views/AppView.js

```diff
@@ -357,15 +357,12 @@ export default class AppViewController {
       <div class="footer">
         <div class="footer-left">
           <a href="https://github.com/chingu-voyages/v47-tier1-team-04" target="_blank">
-            <p class="copyright"><i class="fa-brands fa-github fa-xl"></i> Chingu Team 04 Github</p>
+            <p class="copyright"><i class="fa-brands fa-github fa-xl"></i> Chingu Team 04 Github &copy; 2024</p>
```

Comment:
Instead of static date since we are in a JS template literal writing html we could use ${new Date().getFullYear()}